packages/node_modules/@webex/internal-plugin-conversation/src/share-activity.js

```diff
@@ -251,12 +250,15 @@ const ShareActivity = WebexPlugin.extend({
   /**
    * @param {File} file
    * @param {string} uri
+   * @param {Object} uploadOptions
```

Comment:
Can you outline what the expected format of this object is?

Reply:
I have added additional comment there. Thanks